package-lock.json

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
We don't commit `package-lock` files for now - we didn't decide on it throughout the whole org yet, for more context see https://phabricator.wikimedia.org/T179229